helm/chart/maesh/values.yaml

```diff
@@ -65,6 +65,10 @@ mesh:
   # Added so we can launch on nodes with restrictions
   nodeSelector: {}
   tolerations: []
+  # Additional deployment annotations (e.g. for jaeger-operator sidecar injection)
+  annotations: {}
+  # Additional pod annotations (e.g. for mesh injection or prometheus scraping)
```

Comment:
I'm not sure that these examples are relevant because, we already have `prometheus scraping` annotations and injecting a mesh in a mesh does not make sense, right?